src/Constants.mac.cs.in

```diff
@@ -173,5 +173,7 @@ namespace ObjCRuntime {
 
 		// macOS 10.16
 		public const string UserNotificationsUILibrary = "/System/Library/Frameworks/UserNotificationsUI.framework/UserNotificationsUI";
+		// waiting for it to be fixed in a next beta release: https://github.com/xamarin/maccore/issues/2257
+		// public const string CallKitLibrary = "/System/Library/Frameworks/CallKit.framework/CallKit";
```

Comment:
The issue seems different, affecting only some (not all) API on macOS, right ?
If the file exists (it does on the DTK) then this should not be commented as some tests depends on the fields (and might not execute and hide errors)